Description: fix compilation of libuv on python3 systems
Author: Vasily Smirnov <vasilii.smirnov@globallogic.com>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/deps/libuv-v1.10.1/gyp_uv.py
+++ b/vendor/github.com/armPelionEdge/greasego/deps/src/greaseLib/deps/libuv-v1.10.1/gyp_uv.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 import os
 import platform
